app.js

```diff
@@ -1,30 +1,32 @@
-import express from "express";
-import morgan from "morgan";
-import cors from "cors";
-import path from "path";
-import "dotenv/config";
-import "./db/sequelize.js";
+import express from 'express';
+import morgan from 'morgan';
+import cors from 'cors';
+import path from 'path';
+import 'dotenv/config';
+import './db/sequelize.js';
+import categoriesRouter from './routes/categoriesRouter.js';

```

Comment:
Minor: Could you import the routers together (e.g. healsRouter + categoriesRouter) so they stay next to each other without extra spacing?